fanficfare/defaults.ini

```diff
@@ -2061,6 +2061,9 @@ likes_label:Likes
 reader_input_label:Reader Input
 keep_in_order_tags:true
 
+# Choose whether to include Appendix chapters
```

Comment:
New settings should be added in `calibre-plugin/plugin-defaults.ini` too for Calibre plugin users.